infra/syntax/Json.cpp

```diff
@@ -631,6 +650,28 @@ namespace infra
             return JsonToken::Error();
     }
 
+    JsonToken::Token JsonTokenizer::TryCreateFloatToken(uint64_t integer, bool sign)
+    {
+        if (parseIndex != objectString.size() && objectString[parseIndex] == '-')
+            return JsonToken::Error();
+
+        auto tokenStart = parseIndex;
+
+        while (parseIndex != objectString.size() && std::isdigit(static_cast<unsigned char>(objectString[parseIndex])))
+            ++parseIndex;
+
+        infra::BoundedConstString fractional = objectString.substr(tokenStart, parseIndex - tokenStart);
+        fractional.shrink(nanoValueWidth);
```

Comment:
```suggestion
        if (fractional.size() > nanoValueWidth)
            return JsonToken::Error();
```

Reply:
I am not sure about this one, as in my use case the other side give numbers such as `0.005988637916743755` 
With the shrink it takes `0.005988637`

Reply:
Oh right! Yes, I think you are right